<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WizardSelectNetworks</class>
 <widget class="QWidget" name="WizardSelectNetworks">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>499</width>
    <height>361</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="wizardTittleLabel">
     <property name="text">
      <string>Select the networks you would like to communicate through using VoxOx.</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>70</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QCheckBox" name="chkAIM">
       <property name="text">
        <string>AIM</string>
       </property>
       <property name="icon">
        <iconset resource="../../../../resources/wengophone.qrc">
         <normaloff>:/pics/protocols/aim.png</normaloff>:/pics/protocols/aim.png</iconset>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QCheckBox" name="chkFacebook">
       <property name="text">
        <string>Facebook</string>
       </property>
       <property name="icon">
        <iconset resource="../../../../resources/wengophone.qrc">
         <normaloff>:/pics/protocols/facebook.png</normaloff>:/pics/protocols/facebook.png</iconset>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QCheckBox" name="chkGoogleTalk">
       <property name="text">
        <string>GoogleTalk</string>
       </property>
       <property name="icon">
        <iconset resource="../../../../resources/wengophone.qrc">
         <normaloff>:/pics/protocols/googletalk.png</normaloff>:/pics/protocols/googletalk.png</iconset>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QCheckBox" name="chkICQ">
       <property name="text">
        <string>ICQ</string>
       </property>
       <property name="icon">
        <iconset resource="../../../../resources/wengophone.qrc">
         <normaloff>:/pics/protocols/icq.png</normaloff>:/pics/protocols/icq.png</iconset>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QCheckBox" name="chkJabber">
       <property name="text">
        <string>Jabber</string>
       </property>
       <property name="icon">
        <iconset resource="../../../../resources/wengophone.qrc">
         <normaloff>:/pics/protocols/jabber.png</normaloff>:/pics/protocols/jabber.png</iconset>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QCheckBox" name="chkMSN">
       <property name="text">
        <string>MSN</string>
       </property>
       <property name="icon">
        <iconset resource="../../../../resources/wengophone.qrc">
         <normaloff>:/pics/protocols/msn.png</normaloff>:/pics/protocols/msn.png</iconset>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QCheckBox" name="chkMySpace">
       <property name="text">
        <string>MySpace</string>
       </property>
       <property name="icon">
        <iconset resource="../../../../resources/wengophone.qrc">
         <normaloff>:/pics/protocols/myspaceim.png</normaloff>:/pics/protocols/myspaceim.png</iconset>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QCheckBox" name="chkSkype">
       <property name="text">
        <string>Skype</string>
       </property>
       <property name="icon">
        <iconset resource="../../../../resources/wengophone.qrc">
         <normaloff>:/pics/protocols/skype.png</normaloff>:/pics/protocols/skype.png</iconset>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QCheckBox" name="chkTwitter">
       <property name="text">
        <string>Twitter</string>
       </property>
       <property name="icon">
        <iconset resource="../../../../resources/wengophone.qrc">
         <normaloff>:/pics/protocols/twitter.png</normaloff>:/pics/protocols/twitter.png</iconset>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QCheckBox" name="chkYahoo">
       <property name="text">
        <string>Yahoo</string>
       </property>
       <property name="icon">
        <iconset resource="../../../../resources/wengophone.qrc">
         <normaloff>:/pics/protocols/yahoo.png</normaloff>:/pics/protocols/yahoo.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1" colspan="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>246</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>21</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../../resources/wengophone.qrc"/>
  <include location="../../../../resources/wengophone.qrc"/>
  <include location="../../../../resources/wengophone.qrc"/>
  <include location="../../../../resources/wengophone.qrc"/>
  <include location="../../../../resources/wengophone.qrc"/>
 </resources>
 <connections/>
</ui>
